### PR TITLE
docs(README): add a little starter guide for template ci testing

### PR DESCRIPTION
**Pull request by sassman**
@taurr anything that you would add?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -520,6 +520,63 @@ This last conditional block is simply to ignore the unneeded license files, base
 
 > :warning: Note that `include` and `exclude` are still mutually exclusive even if they are in different, but included, conditional sections.
 
+## Useful for template authors
+
+> Available since version [0.9.0]
+
+As a template author you're probably concerned about successful builds of your template?
+
+Imagine a couple of month after your first template release, some new versions of any dependencies would break your template, and you would not even be aware of it?
```
Comment (taurr) [inline README.md:529]: month => months... plural :-)

File: README.md [hunk 2]
```diff
@@ -520,6 +520,63 @@ This last conditional block is simply to ignore the unneeded license files, base
...
+    └── build.yml
+```
+
+The content of `build.yml` as a paste template:
+```yaml
+name: Build Template
+on:
+  schedule:
+    - cron: '0 18 * * 5'
+  push:
```
Comment (taurr) [inline README.md:548]: please also add so that it can be triggered manually

File: README.md [hunk 3]
```diff
@@ -520,6 +520,63 @@ This last conditional block is simply to ignore the unneeded license files, base
...
 
+## Useful for template authors
+
+> Available since version [0.9.0]
+
+As a template author you're probably concerned about successful builds of your template?
+
+Imagine a couple of month after your first template release, some new versions of any dependencies would break your template, and you would not even be aware of it?
+
+The answer to this question is a vital build pipeline for your template project. This challenge got now much simpler to solve with the new official [GitHub Action cargo-generate][gh/action].
```
Comment (taurr) [inline README.md:531]: Didn't know of the md link style with 2 following `[xxx][yyy]`... only the normal `[xxx](link)` or `[xxx]` and then later `[xxx]: link`

Guess I now know how to rename the link in the latter style... cool 👍🏽

**Review by taurr — APPROVED** [review]
Looks good to me.

And definitely worth mentioning the GH action in the README.

File: README.md (hunk 2 above)
Comment (sassman) [reply README.md:548]: very good idea

File: README.md (hunk 1 above)
Comment (sassman) [reply README.md:529]: now you know who is also a very comprehensive writer..

**Pull request merged**